### ISSUE TITLE
[Keyboard Navigation - Error message]: The 'Move to next problem' and 'move to previous problem' buttons are not accessible through keyboard while navigating using tab key.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#28125]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.80.1
- OS Version: Windows_NT x64 10.0.22621

Steps to Reproduce:

1. Type something that can trigger error messages.
2. The 'Move to next problem' and 'move to previous problem' buttons are not accessible through keyboard while navigating using tab key. The root cause seems to be the `isKeyboardFocusable=false` attribute
![image](https://github-com/microsoft/vscode/assets/104450101/b5b69d98-e453-4bf8-ab61-96ce7a582a95)
